src/packages/@workspaces/features/rest-explorer/components/response-status/ResponseStatus.svelte

```diff
@@ -25,20 +25,20 @@
             style="font-size: 12px;"
           >
             <span
-              class="ellipsis"
+              class="ellipsis d-flex align-items-center"
               style="color:{response.status === ResponseStatusCode.OK ||
               response.status === ResponseStatusCode.CREATED
-                ? 'var(--success-color)'
+                ? 'var(--request-get)'
```

Comment:
instead of this use color code as per our nomenclature as we have used in other places.